packages/amplify-graphql-auth-transformer/src/utils/index.ts

```diff
@@ -56,6 +65,17 @@ export const ensureAuthRuleDefaults = (rules: AuthRule[]) => {
   }
 };
 
+export const getAuthDirectiveRules = (authDir: DirectiveWrapper): AuthRule[] => {
```

Comment:
should probably merge this with `ensureAuthRuleDefaults`

Reply:
merged